docs/sources/guides/whats-new-in-v7-2.md

```diff
@@ -157,3 +158,7 @@ Check out [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELO
 In collaboration with Microsoft, we have improved the usability of our ADX datasource plugin by adding a visual query builder. The goal is to make it easier for users, regardless of their previous knowledge of writing KQL (Kusto Query Language) queries, to query and visualize their data.
 
 {{< docs-imagebox img="/img/docs/v72/adx-ds.png" max-width="800px" caption="ADX visual query builder" >}}
+
+### X-Ray data source plugin
```

Comment:
I see the problem. @zoltanbedi, please update the TOC at the top to include the missing items, including the two headings above your entry.

Reply:
Added the missing titles.